phpstan.neon.dist

```diff
@@ -20,14 +20,10 @@ parameters:
 		- tests/Fixtures/app/var/
 		- tests/Fixtures/Symfony/Maker
 		# BC layer
-		- src/deprecated_interfaces.php
 		- tests/Fixtures/TestBundle/BrowserKit/Client.php
 		# The Symfony Configuration API isn't good enough to be analysed
 		- src/Symfony/Bundle/DependencyInjection/Configuration.php
-		# Phpstan runs on phpunit > 9, a signature changed in this file
-		- src/Symfony/Bundle/Test/Constraint/ArraySubsetLegacy.php
 		# Imported code (temporary)
-		- src/Symfony/Bundle/Test/BrowserKitAssertionsTrait.php
 		- tests/Symfony/Bundle/Test/WebTestCaseTest.php
```

Comment:
This line needs to be checked

Reply:
This PR is a WIP then?

Reply:
We can do it in this PR or in a new one. I think this class can be removed but I'm not sure.

Reply:
I will do it in another PR :slightly_smiling_face:

Reply:
Done here: https://github.com/api-platform/core/pull/4866